blackbox/docs/src/doc_tests/tests.py

```diff
@@ -191,6 +191,17 @@ def setUpLocations(test):
     locations_file = get_abspath("locations.json")
     _execute_sql("""copy locations from '{0}'""".format(locations_file))
     _execute_sql("""refresh table locations""")
+    import_failures_dir = '/tmp/import_data/locations_with_failure'
+    if not os.path.isdir(import_failures_dir):
+        os.mkdir(import_failures_dir)
```

Comment:
Minor: Could use `os.makedirs(..., exist_ok=True)`, avoids the "race condition" of the not-exists-then-create. (Though in this case it doesn't matter)